### PR TITLE
Cluster Autoscaler AKT defaults

### PR DESCRIPTION
**Pull request by njtran**
*Issue #, if available:*

Made it so that you can specify resource limits for the CA, and added defaults.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: eksconfig/clusterautoscaler.go [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
-	Image          string        `json:"image,omitempty"`
-	CloudProvider  CloudProvider `json:"cloudProvider,omitempty"`
-	MinNodes       int           `json:"minNodes,omitempty"`
-	MaxNodes       int           `json:"maxNodes,omitempty"`
-	ScaleDownDelay string        `json:"scaleDownDelay,omitempty"`
+	Image          string      	 				`json:"image,omitempty"`
+	CloudProvider  CloudProvider 				`json:"cloudProvider,omitempty"`
+	MinNodes       int           				`json:"minNodes,omitempty"`
+	MaxNodes       int           				`json:"maxNodes,omitempty"`
+	Resources      corev1.ResourceRequirements  `json:"resources,omitempty"`
```
Comment (ellistarn) [inline eksconfig/clusterautoscaler.go:14]: These json comments should be aligned. Do you have a go formatter installed?

File: eksconfig/clusterautoscaler.go [hunk 2]
```diff
@@ -39,6 +45,12 @@ func (spec *ClusterAutoscalerSpec) Default(cfg *Config) {
 	if spec.MaxNodes == 0 {
 		spec.MaxNodes = 100
 	}
+	if spec.Resources.Requests.Cpu().IsZero() {
```
Comment (ellistarn) [inline eksconfig/clusterautoscaler.go:48]: It's good to have these fields in the spec, but they're not actually used anywhere. Hint: they need to be in the gotemplate.

File: eksconfig/clusterautoscaler.go (hunk 1 above)
Comment (gyuho) [reply eksconfig/clusterautoscaler.go:14]: @njtran You can run `./hack/fmt.sh`?

File: eks/clusterautoscaler/clusterautoscaler.go [hunk 3]
```diff
@@ -89,6 +91,12 @@ func (c *ClusterAutoscaler) buildCommand() string {
 	return string(json)
 }
 
+func (c *ClusterAutoscaler) buildResources() string {
+	json, _ := json.Marshal(c.Config.Spec.ClusterAutoscaler.Resources)
```
Comment (ellistarn) [inline eks/clusterautoscaler/clusterautoscaler.go:95]: These are actually already included in the gotemplate from c.Config.Spec.ClusterAutoscaler on line 36.

Comment (njtran) [reply eks/clusterautoscaler/clusterautoscaler.go]: Got it, changed.

**Comment by ellistarn** [comment]
Nice. Just a weird formatting issue

**Pull request merged**